homeassistant/components/climate/nest.py

```diff
@@ -40,9 +40,16 @@ def __init__(self, structure, device, temp_unit):
         self.structure = structure
         self.device = device
         self._fan_list = [STATE_ON, STATE_AUTO]
+
+        """Not all nest devices support cooling and heating remove unused"""
```

Comment:
Use `#` at the start of a line for a comment. Only make comments with triple quotes as first line of a method or class, as that's the convention to document things.